include/aikido/constraint/uniform/SO2UniformSampler.hpp

```diff
@@ -31,6 +32,9 @@ class SO2Sampleable : public constraint::Sampleable
   std::unique_ptr<util::RNG> mRng;
 };
 
+using SO2Sampleable = SO2UniformSampler;
```

Comment:
I think we should either add a deprecation warning or just not have this old name at all. Since I don't think there's anything using this yet, I vote for the latter. :smiling_imp: